src/main/java/com/epam/reportportal/junit5/ReportPortalExtension.java

```diff
@@ -266,6 +264,33 @@ private void startTestItem(ExtensionContext context, String type) {
 		startTestItem(context, Collections.emptyList(), type, null);
 	}
 
+	private String getCodeRef(Method method) {
+		return method.getDeclaringClass().getCanonicalName() + "." + method.getName();
+	}
+
+	private static String appendSuffixIfNotEmpty(String str, String suffix) {
+		return str + (suffix.isEmpty() ? "" : "$" + suffix);
+	}
+
+	private String getCodeRef(ExtensionContext context, String currentCodeRef) {
+		Optional<Method> testMethod = context.getTestMethod();
+		if (testMethod.isPresent()) {
```

Comment:
Probably can be replaced with ofNullable().map().orElseGet() here

Reply:
OK